cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/NativeUtils.java

```diff
@@ -91,28 +96,29 @@ public static void createReflectConfig(Path nativeConfigPath, Package currentPac
                             new String[]{"java.lang.String[]", "java.nio.file.Path[]", "java.lang.String"}
                     )
             );
-
-            ReflectConfigClass testTestExecuteGenerated = new ReflectConfigClass(getQualifiedClassName(org, name,
-                    version, MODULE_EXECUTE_GENERATED + tally));
-
-            testTestExecuteGenerated.addReflectConfigClassMethod(
-                    new ReflectConfigClassMethod(
-                            "__execute__",
-                            new String[]{
-                                    "io.ballerina.runtime.internal.scheduling.Strand",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString",
-                                    "io.ballerina.runtime.api.values.BString"
-                            }
-                    )
-            );
+            ReflectConfigClass testTestExecuteGenerated = new ReflectConfigClass("");
```

Comment:
The variable name can be improved here. The IDE would suggest the variable name.
```suggestion
            ReflectConfigClass reflectConfigClass = new ReflectConfigClass("");
```

Reply:
There are few more `ReflectConfigClass` instances are created in same `NativeUtils.java`. So, The naming is based on how it is utilised(Each instance represents a class in `ReflectionConfig`).